tracing-opentelemetry/src/subscriber.rs

```diff
@@ -21,14 +29,23 @@ const SPAN_NAME_FIELD: &str = "otel.name";
 const SPAN_KIND_FIELD: &str = "otel.kind";
 const SPAN_STATUS_CODE_FIELD: &str = "otel.status_code";
 const SPAN_STATUS_MESSAGE_FIELD: &str = "otel.status_message";
+const INSTRUMENTATION_NAME: &str = "tracing/tracing-opentelemetry";
+
+struct Instruments_(Arc<RwLock<Instruments>>);
 
+struct MetricsHandler {
+    _push_controller: PushController,
```

Comment:
I put underscore because it was unused right now (only needed to create the `Meter`), but I think we might want to require this to be passed in, so that we can make use of it later on in follow-up improvements 🤔